apps/andi/test/integration/andi_web/live/edit_organization_live_view_test.exs

```diff
@@ -107,6 +107,13 @@ defmodule AndiWeb.EditOrganizationLiveViewTest do
 
       where(title: ["", "!@#$%"])
     end
+
+    test "user has the ability to add an organization on the organizations page", %{conn: conn} do
```

Comment:
this seems like the wrong test file, I would maybe break up the tests into a new module testing only the `organization_live_view` stuff

Reply:
Also, are there any tests around the uniqueness of the orgName in this test file? I feel like that error will always come up now because of the `or` in the uniqueness check above. So it might be worth putting one in here

Reply:
This is no `organization_live_view_test` so i figured it would be ok to put this here